src/modules/app/load.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * Module Name: App
+ * Experimental: No
+ *
+ * @since   1.0.0
+ * @package wordlift
+ */
+
+use Wordlift\Modules\App\Plugin_App;
+use Wordlift\Modules\Common\Symfony\Component\Config\FileLocator;
+use Wordlift\Modules\Common\Symfony\Component\DependencyInjection\ContainerBuilder;
+use Wordlift\Modules\Common\Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
+
+// Autoloader for plugin itself.
```

Comment:
@naveen17797 this is loaded only when we're in admin right, `is_admin()`? I want to be sure we don't load this for every client call.